tardis/io/model/parse_radiation_field_configuration.py

```diff
@@ -167,15 +167,14 @@ def calculate_geometric_dilution_factor(geometry):
     np.array
         The dilution factors for the inout geometry
     """
-    return 0.5 * (
+    value = (
         1
-        - np.sqrt(
-            1
-            - (
-                geometry.r_inner[geometry.v_inner_boundary_index] ** 2
-                / geometry.r_middle**2
-            )
-            .to(1)
-            .value
+        - (
+            geometry.r_inner[geometry.v_inner_boundary_index] ** 2
+            / geometry.r_middle**2
         )
+        .to(1)
+        .value
     )
+    value = np.clip(value, 0, None)
```

Comment:
What warning does this resolve?

Reply:
RuntimeWarning: invalid value encountered in sqrt
    - np.sqrt(